tools/renderer_html.py

```
@@ -0,0 +1,582 @@
+            packet["video"]["canvas"] = f"{config['viewport_width']}x{config['viewport_height']}"
+            
+            # Get duration
+            duration = get_video_duration(Path(out_mp4))
+            if duration > 0:
+                packet["video"]["duration_s"] = duration
+            
+            changed = True
+            rendered_count += 1
+            logger.info(f"Rendered video for {story_id}: {out_mp4} ({packet_render_time:.0f}ms)")
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)